### PR TITLE
docs: Add a draft Readme to CMS (FI-16)

### PR DESCRIPTION
**Pull request by elien2016**
## Describe your changes
updated README

## Issue ticket number and link
FI-16
https://catting.atlassian.net/browse/FI-16?atlOrigin=eyJpIjoiYWJlZTgxODJkYTBhNDY3ZGExMmZmZTViYzc2MmQ5MmMiLCJwIjoiaiJ9

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by ybw0014** [comment]
We have updated the secret names (https://github.com/Charlie059/Flow.IO/pull/7#issue-2061112646), so it is better to update them in the sample as well.
I think we can add a `.env.example` under `@core/cms` so it wont take up space in readme.

**Comment by Charlie059** [comment]
@ybw0014, @elien2016 Thank you all!

**Comment by elien2016** [comment]
> We have updated the secret names ([#7 (comment)](https://github.com/Charlie059/Flow.IO/pull/7#issue-2061112646)), so it is better to update them in the sample as well. I think we can add a `.env.example` under `@core/cms` so it wont take up space in readme.

Updated. Lmk if there's anything else that needs changing:)

File: @core/cms/.env.example
Comment (Charlie059) [inline @core/cms/.env.example:1]: Not sure if this can pass gitguardian, but let me try

**Review by Charlie059 — APPROVED** [review]

**Pull request merged**